src/background/telemetry/telemetry-client-provider.ts

```diff
@@ -7,15 +7,15 @@ import { DateProvider } from '../../common/date-provider';
 import { generateUID } from '../../common/uid-generator';
 import { ApplicationBuildGenerator } from '../application-build-generator';
 import { InstallDataGenerator } from '../install-data-generator';
-import { LocalStorageData } from '../storage-data';
+import { InstallationData } from '../installation-data';
 import { AppInsightsTelemetryClient } from './app-insights-telemetry-client';
 import { ApplicationTelemetryDataFactory } from './application-telemetry-data-factory';
 import { NullTelemetryClient } from './null-telemetry-client';
 import { TelemetryClient } from './telemetry-client';
 import { TelemetryLogger } from './telemetry-logger';
 
 export const getTelemetryClient = (
-    userData: LocalStorageData,
+    installationData: InstallationData,
```

Comment:
nitpick: it's probably not worth to actually do this but... the signature change of this function is self-contain and makes sense in it's own... so this change could be it's own small PR.